client/src/documentLinkProvider.ts

```diff
@@ -57,8 +58,8 @@ export class BitbakeDocumentLinkProvider implements vscode.DocumentLinkProvider
     }
     let foundDirs: string[] = []
     try {
-      foundDirs = foundDirs.concat(find.dirSync(pnDir))
-      foundDirs = foundDirs.concat(find.dirSync(filesDir))
+      if (fs.existsSync(pnDir)) foundDirs = foundDirs.concat(find.dirSync(pnDir))
```

Comment:
since you added the check, we probably don't need the try-catch block anymore. It was the only place that I could think of that could throw an error.

Reply:
existSync can still throw, or dirSync, if an error happens?
For instance if the dir exist, but we don't have permission to read it.

Reply:
oh yes, you got a point. We can keep it.